### ISSUE TITLE
FB changed their UI. 

### ISSUE THREAD

**Issue by AkibSadmanee** [lalongooo/selenium-fb-group-poster#8]
FB changed their UI. So the xpaths are not in function anymore.

**Comment by iamyorchdev** [comment]
Yes, I'm aware of that. However, these days I'm short of time. I'll update the script later.
#StayHome #StaySafe

**Comment by adarfrenkel** [comment]
Dear lalongooo,
Your script is awesome! can you please adapt it to the new UI?
